src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/InMemory/SubscriptionsOrchestratorJobTests.cs

```diff
@@ -0,0 +1,234 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Hl7.Fhir.Serialization;
+using Hl7.Fhir.Utility;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features.Context;
+using Microsoft.Health.Fhir.Core.Features.Definition;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.Fhir.Core.Features.Routing;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Features.Search.Access;
+using Microsoft.Health.Fhir.Core.Features.Search.Converters;
+using Microsoft.Health.Fhir.Core.Features.Search.Expressions.Parsers;
+using Microsoft.Health.Fhir.Core.Features.Search.InMemory;
+using Microsoft.Health.Fhir.Core.Features.Search.SearchValues;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.Subscriptions.Models;
+using Microsoft.Health.Fhir.Subscriptions.Operations;
+using Microsoft.Health.Fhir.Subscriptions.Persistence;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.JobManagement;
+using Newtonsoft.Json;
+using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Search.InMemory
+{
+    public class SubscriptionsOrchestratorJobTests : IAsyncLifetime
```

Comment:
I think there are two options for the tests since they don't yet work in all versions of FHIR:

1) use [SkippableFact] and Skip if ModelInfoProvider.Version != FhirSpecification.R4
2 OR Move to the Microsoft.Health.Fhir.Core.R4.Tests project

@feordin thoughts?